client.go

```diff
@@ -33,6 +34,11 @@ type Client struct {
 	debugLogFunc LogFunc
 }
 
+// ExportedFuncWithBadComent is a bad comment
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function ExportedFuncWithBadComment should be of the form "ExportedFuncWithBadComment ..." (golint)